api/v1alpha4/conditions_consts.go

```diff
@@ -20,6 +20,8 @@ import clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
 
 // AzureCluster Conditions and Reasons.
 const (
+	// NetworkInfrastructureReadyCondition reports of current status of cluster infrastructure.
+	NetworkInfrastructureReadyCondition clusterv1.ConditionType = "NetworkInfrastructureReady"
```

Comment:
re-added so that the cluster doesn't show as Ready until all services are done reconciling (might remove once each service has its own condition in place to use as summary)